.pre-commit-config.yaml

```diff
@@ -1,10 +1,10 @@
 repos:
-  - repo: https://github.com/gitguardian/ggshield
-    rev: v1.13.0
-    hooks:
-      - id: ggshield
-        language_version: python3
-        stages: [commit]
+  # - repo: https://github.com/gitguardian/ggshield
```

Comment:
Thanks @NicholasTurner23 , could you share some notes about this change of the pre-commit configuration?